spec/classes/dhcp_spec.rb

```diff
@@ -4,6 +4,7 @@
     {
       'dnsdomain'           => ['sampledomain.com', '1.1.1.in-addr.arpa'],
       'nameservers'         => ['1.1.1.1'],
+      'nameservers_ipv6'    => ['1:5ee:bad:c0de'],
```

Comment:
This is not a valid IPv6 address. Try adding an extra ```:``` and it should at least pass validation if we decide to add it.